.github/workflows/pull-request.yml

```diff
@@ -7,7 +7,11 @@ on:
 
 jobs:
   build-and-test:
-    runs-on: windows-latest
+    runs-on: ${{ matrix.os }}
+    strategy:
```

Comment:
what is the plan to migrate the rolling-build workflow to linux as well and queue a manually triggered run on that one? I'm ok to make that as a separate PR if this unduly delays this PR

Reply:
The changes for the rolling build are fairly simple, but as-is we'll need to set `max-parallel: 1` since the arguments are the same, and one OS's run could delete the test org while the other is still using it.

Looks like the rolling build is already failing on the main branch.  Should I go ahead and add the linux run as well, or wait until the rolling build is back in a good state?

Reply:
let's wait with adding linux to rolling until the win build is good. The error is weird, the envId doesn't resolve, but looking from e.g. my local pac admin list, it does under the given env guid (using latest pac CLI from master).